src/Disqord.Rest.Api/Routing/Route.Static.cs

```diff
@@ -209,6 +209,19 @@ public static class Guild
             public static readonly Route ModifyMemberVoiceState = Patch("guilds/{0:guild_id}/voice-states/{1:user_id}");
         }
 
+        public static class GuildEvents
+        {
+            public static readonly Route GetGuildEvents = Get("guilds/{0:guild_id}/events");

```

Comment:
You can remove the redundant `Guild` from the routes.
```suggestion
            public static readonly Route GetEvents = Get("guilds/{0:guild_id}/events");
```